### PR TITLE
Adding log level

### PR DESCRIPTION
**Pull request by agoncal**
I think it's important to mention all the log levels. There are so many loggers that we might get confused (eg. TRACE vs FINEST, etc.)

### CODE REVIEW

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -54,6 +54,21 @@ The root logger category is handled separately, and is configured via the follow
...
 
+=== Log levels
+
+There are several log levels you can use:
+
+[cols="<m,<5",options="header"]
+|===
+|Level|Description
+|FATAL|For events that indicate a critical service failure
+|ERROR|To indicate a disruption in a request or the ability to service a request
```
Comment (gastaldi) [inline docs/src/main/asciidoc/logging.adoc:65]:
```suggestion
|ERROR|To indicate a disruption in a request or the inability to service a request
```

File: docs/src/main/asciidoc/logging.adoc [hunk 2]
```diff
@@ -54,6 +54,21 @@ The root logger category is handled separately, and is configured via the follow
...
+
+[cols="<m,<5",options="header"]
+|===
+|Level|Description
+|FATAL|For events that indicate a critical service failure
+|ERROR|To indicate a disruption in a request or the inability to service a request
+|WARN|May indicate a non-critical service error
+|INFO|For crucial related information
+|DEBUG|For messages that convey extra information
+|TRACE|Log low-level priority messages
```
Comment (dmlloyd) [inline docs/src/main/asciidoc/logging.adoc:69]:
```suggestion
|FATAL|A critical service failure/complete inability to service requests of any kind
|ERROR|A significant disruption in a request or the inability to service a request
|WARN|A non-critical service error or problem that may not require immediate correction
|INFO|Service lifecycle events or important related very-low-frequency information
|DEBUG|Messages that convey extra information regarding lifecycle or non-request-bound events which may be helpful for debugging
|TRACE|Messages that convey extra per-request debugging information that may be very high frequency
```

**Review by dmlloyd — COMMENTED** [review]
I'd refine the wording a little bit.

**Comment by gastaldi** [comment]
Can you squash your commits? Thanks!

**Review by gastaldi — APPROVED** [review]
I squashed the commits for you. LGTM

**Comment by agoncal** [comment]
Oups, sorry, not fast enough. Thanks @gastaldi

**Pull request merged**